pkg/cmd/server/api/types.go

```diff
@@ -65,6 +65,8 @@ type MasterConfig struct {
 	ImageConfig ImageConfig
 
 	PolicyConfig PolicyConfig
+
+	SchedulerConfigFile string
```

Comment:
Let's make a separate stanza for this.  Almost nothing has remained single argument and creating the stanza makes sure that the related options come out together in the serialization since it's straight alphabetical.